src/Console/Console.php

```diff
@@ -4,9 +4,6 @@
 
 namespace Tempest\Console;
 
-use Tempest\Container\InitializedBy;
-
-#[InitializedBy(ConsoleInitializer::class)]
 interface Console extends ConsoleInput, ConsoleOutput
```

Comment:
Kinda unrelated to this PR, but do we actually need this interface?

Reply:
Yes, for convenience. Most of the time, you're dealing with both input and output, and having one interface that handles both is much nicer.